src/components/Sing/ScoreSequencer.vue

```diff
@@ -776,6 +820,46 @@ const onMouseUp = (event: MouseEvent) => {
   nowPreviewing.value = false;
 };
 
+const quadSelect = async () => {
+  const quadSelectHitboxElement = quadSelectHitbox.value;
+  if (!quadSelectHitboxElement) {
+    throw new Error("quadSelectHitboxElement is null.");
+  }
+  isQuadSelecting.value = false;
+  const left = Math.min(quadSelectStartX.value, cursorX);
+  const top = Math.min(quadSelectStartY.value, cursorY);
+  const width = Math.abs(cursorX - quadSelectStartX.value);
+  const height = Math.abs(cursorY - quadSelectStartY.value);
+  const startTicks = baseXToTick(
+    (scrollX.value + left) / zoomX.value,
+    tpqn.value
+  );
+  const endTicks = baseXToTick(
+    (scrollX.value + left + width) / zoomX.value,
+    tpqn.value
+  );
+  const startNoteNumber = baseYToNoteNumber(
+    (scrollY.value + top) / zoomY.value
+  );
+  const endNoteNumber = baseYToNoteNumber(
+    (scrollY.value + top + height) / zoomY.value
+  );
```

Comment:
画面は左上が(0, 0)ですが、NoteNumberは一番下が0なので、変数名は逆の方が良いかも？
```suggestion
  const endNoteNumber = baseYToNoteNumber(
    (scrollY.value + top) / zoomY.value
  );
  const startNoteNumber = baseYToNoteNumber(
    (scrollY.value + top + height) / zoomY.value
  );
```